packages/main/src/GroupHeaderListItem.ts

```diff
@@ -17,11 +17,8 @@ import groupheaderListItemCss from "./generated/themes/GroupHeaderListItem.css.j
  * The <code>ui5-li-groupheader</code> is a special list item, used only to separate other list items into logical groups.
  *
  * @constructor
- * @author SAP SE
- * @alias sap.ui.webc.main.GroupHeaderListItem
- * @extends sap.ui.webc.main.ListItemBase
- * @tagname ui5-li-groupheader
- * @implements sap.ui.webc.main.IListItem
+ * @extends ListItemBase
+ * @implements {IListItem}
  * @public
  */
```

Comment:
- Make the CustomListItem class to explicitly implements IListItem in TypeScript
- Remove `@type` tag
- Move default slot comment to class comment (check how to describe unnamed slots)